src/test/online_localizer_test.cpp

```diff
@@ -0,0 +1,62 @@
+
+#include "database/cost_matrix_database.h"
+#include "features/ifeature.h"
+#include "localization_protos.pb.h"
+#include "online_localizer/online_localizer.h"
+#include "relocalizers/irelocalizer.h"
+#include "successor_manager/successor_manager.h"
+#include "test_utils.h"
```

Comment:
Do you need all of these includes?

Reply:
It does compile without the includes. But I would like to keep includes since I am using the types from those files.

Reply:
This is fine. The best practice on this is "include what you see", so if you do use stuff from all of these files they must be included. Thanks!